### ISSUE TITLE
 How to duplicate retroarch content on the same monitor? (WITHOUT SHADERS)

### ISSUE THREAD

**Issue by Pietro1994** [libretro/RetroArch#10796]
Hi. 

I wanted to have info on this topic: 
the **duplication (cloning) of the game content in retroarch.**
I was wondering if there was a function available (in gl.c or some nearby library) that would allow you to clone the game content on the same screen.
Basically I would like to create a kind of '_**cocktail-cabinet**_' without the use of the _shader_. I was wondering if there was a function in retroarch or libretro that would allow me to create a duplicate of the content.

Thanks in advance.

**Comment by hizzlekizzle** [comment]
Yeah, you could do it without a shader just through GL stuff, but then you also need to add menu entries, etc. We generally try to handle as much video-related stuff with shaders as possible because it's more flexible and easier than making static options.

**Comment by Pietro1994** [comment]
Thanks for your response and availability.

If I were to duplicate the contents of the screen, what is the function (the name) of the GL library that I should call? (example: clone_screen, duplicate_display, etc ...).
Menu items is not a problem, I know how to do it.

Thanks in advance.

**Comment by hizzlekizzle** [comment]
I'm not sure exactly where this would go. Probably in the individual video driver implementations.
